app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,68 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.ui.theme.spacing
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+data class MovieCardViewState(
+    val imageUrl: String?,
+    val title: String,
+    val isFavorite: Boolean
+)
+
+@Composable
+fun MovieCard(
+    modifier: Modifier,
+    movieCardViewState: MovieCardViewState,
+    onClick: () -> Unit
```

Comment:
Removed unused parameter.

Reply:
`onFavoriteButtonClicked` was needed here but you had to use it. 
Reference to that lambda should be passed to FavoriteButton so it has like click functionality. 
Currently FavoriteButton doesn't do anything when you click on it.